plugins/inputs/t128_graphql/t128_graphql.go

```diff
@@ -138,22 +140,38 @@ func (plugin *T128GraphQL) checkConfig() error {
 	return nil
 }
 
-//Gather takes in an accumulator and adds the metrics that the Input gathers
+// Gather takes in an accumulator and adds the metrics that the Input gathers
 func (plugin *T128GraphQL) Gather(acc telegraf.Accumulator) error {
+
+	processedResponses, _ := plugin.SharedQueryCreation(acc)
+	if processedResponses != nil {
+		for _, processedResponse := range processedResponses {
+			acc.AddFields(
+				plugin.CollectorName,
+				processedResponse.Fields,
+				processedResponse.Tags,
+			)
+		}
+	}
+
+	return nil
+}
+
+func (plugin *T128GraphQL) SharedQueryCreation(acc telegraf.Accumulator) ([]*ProcessedResponse, error) {
```

Comment:
`SharedQueryCreation` -> `MakeRequest`. Also, public functions should have comments. That would be a good place to indicate this can be used by other plugins.